sources/TileDB.CSharp/Marshalling/MarshaledStringOut.cs

```diff
@@ -0,0 +1,21 @@
+using System;
+using System.Text;
+
+namespace TileDB.Interop
+{
+    internal unsafe struct MarshaledStringOut
```

Comment:
I made `MarshaledStringOut` a static class. Instead of creating objects of this type, we directly deal with `sbyte*` and call `MarshaledStringOut.GetStringFromNullTerminated` with that pointer.

Reply:
👍 Thanks!